docs/README.md

```diff
@@ -1,7 +1,198 @@
 # Return App V3 <sup>BETA</sup>
 
-Return app v3 is still in BETA. Docs are WIP.
+Return app v3 is still in BETA. v2 will no longer be supported so please do not install it. 
+ Docs are WIP.
 
+## Description
+
+The **Return App** gives merchants the ability to manage the Return Request Process on their store. 
+
+## Features
+
+### Admin Return Request List
+In this section of the merchant's admin, merchants are capable of visualizing and managing all the return requests created by their customers. 
+### Admin Return Settings
+In this section of the merchant's admin, merchants control what are the conditions of allowing the return process for a given item or items per customer. 
+- **Max Days**: when an order creation date is older than the Max Days to return, user won't be able to select that order for a return
+- **Terms and Conditions**: link to the Terms and conditions offered by the store. (not provided by the app)
+- **Excluded categories**: List of categories to be excluded for the return process. Any item that belongs to any of these categories, will not be allowed to be returned.
```

Comment:
How about adding that the store user will see a message stating that that item is not allowed to be returned?